### DISCUSSION TITLE
zh_CN/stable/ops/faq/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2889] [Announcements]
# zh_CN/stable/ops/faq/

https://kube-ovn.readthedocs.io/zh_CN/stable/ops/faq/

**Comment by zhongyong** [comment]
aarch 上面的欧拉操作系统22.03 sp1 lts这个问题怎么解决呢？

**Reply by oilbeater** [reply]
哪个问题

**Reply by zhongyong** [reply]
鲲鹏服务器部署欧拉22.03 sp1 lts ，2台机器一个master一台node。  容器跨节点访问不稳定，需要ethtool -K eth0 tx off 这样解决，

**Reply by oilbeater** [reply]
这样设置后还有别的问题么

**Reply by zhongyong** [reply]
这样设置后解决了问题，但是也让插件稍了很多功能呀

**Reply by oilbeater** [reply]
这个设置会带来性能下降，对功能没有影响。问题的根源在内核和网卡驱动上，如果要根本解决需要联系操作系统的人来处理

**Reply by zhongyong** [reply]
谢谢明白了，我开始描述错误了，性能下降功能没影响
